frontend/components/Map.jsx

```diff
@@ -0,0 +1,117 @@
+import 'leaflet/dist/leaflet.css'
+import React, { useState } from 'react'
+import { GeoJSON, MapContainer, useMap } from 'react-leaflet'
+import States from "./map-assets/gz_2010_us_040_00_5m.json"
+import Counties from "./map-assets/gz_2010_us_050_00_5m.json"
+import styles from './map.module.css'
+
+export default function Map() {
+
+    let center = [35, -90]
+    let startingZoom = 4
+    const [selectedState, setSelectedState] = useState(1)
+
+    return (
+        <div className={styles.mappage} >
+            <Header />
+            <div className={styles.mapframe}>
+                <MapContainer
+                    center={center}
+                    zoom={startingZoom}
+                    className={styles.mapcontainer}
+                >
+                    <GeoDataStates selectedState={selectedState} setSelectedState={setSelectedState} counties={Counties} />
+                </MapContainer>
+            </div>
+        </div>
+    )
+}
+
+export function GeoDataStates() {
+
+    const map = useMap()
+
+    const [gState, setGState] = useState()
```

Comment:
I've always heard the advice to avoid abbreviations in variable names as it helps with clarity (like as a first-time reader I don't know what 'G' stands for)